bigtable/tests/unit/test_table.py

```diff
@@ -1628,3 +1746,12 @@ def _ColumnFamilyPB(*args, **kw):
         table_pb2 as table_v2_pb2)
 
     return table_v2_pb2.ColumnFamily(*args, **kw)
+
+
+def _ClusterStatePB(replication_state):
+    from google.cloud.bigtable_admin_v2.proto import (
+        table_pb2 as table_v2_pb2)
+
+    return table_v2_pb2.Table.ClusterState(
+        replication_state=replication_state
+    )
```

Comment:
Remove this class (we should be removing them across the board).

Reply:
I agree with you. I have used it as:

response_pb = _TablePB(
            cluster_states={'cluster-id1': _ClusterState(INITIALIZING),
                            'cluster-id2': _ClusterState(
                                PLANNED_MAINTENANCE),
                            'cluster-id3': _ClusterState(READY),
                            },
        )

Am I missing something here?

Reply:
@vikas-jamdar 

> vikas-jamdar 

Nope, I misread the `_ClusterStatePB` bit as mocking a protobuf message, rather than returning one.  My confusion was increased by the use of a private class (`_ClusterState`) for the values returned in the mapping by `Table.get_cluster_states`.